src/python/WMCore/Services/Requests.py

```diff
@@ -255,6 +255,13 @@ def makeRequest_httplib(self, uri=None, data={}, verb='GET',
                 # socket/httplib really screwed up - nuclear option
                 conn.connections = {}
                 raise socket.error('Error contacting: %s: %s' % (self.getDomainName(), msg))
+        except Exception as ex:
+            if "Unable to find the server at" in str(ex):
```

Comment:
I see I missed that. Then I think we should catch that exception instead of checking the message string.